basiclti/basiclti-tool/src/java/org/sakaiproject/blti/tool/LTIAdminTool.java

```diff
@@ -205,6 +434,30 @@ public String buildToolSitePanelContext(VelocityPortlet portlet, Context context
 		context.put("isAdmin",new Boolean(ltiService.isAdmin()) );
 		context.put("getContext",toolManager.getCurrentPlacement().getContext());
 		
+		//put velocity date tool in the context
+		context.put("dateTool", new DateTool());
+		DateFormat df = DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT, rb.getLocale());
+		if (df instanceof SimpleDateFormat)
+		{
+			SimpleDateFormat sdf = (SimpleDateFormat)df;
+			context.put("dateToolFormat",sdf.toLocalizedPattern());
```

Comment:
Yes, but I don't like to manually format dates in the java side. If we do that, we need to include in the context (or in this case, in the content object) the date and the formatted date. Now is just a field, but if in the future we add more date fields we will need to do this process for each one.

If we put the dateTool in the context, we can format the dates in the velocity file.